.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,239 @@
+name: Integration Tests
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
+      - develop
+      - release*
```

Comment:
Mind adding `auto`, `rollup` and `try` ?

Reply:
Done!